test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -1420,10 +1420,12 @@ func addKubeSystemPdbs(f *framework.Framework) (func(), error) {
 		min_available int
 	}
 	pdbsToAdd := []pdbInfo{
-		{label: "kube-dns-autoscaler", min_available: 1},
 		{label: "kube-dns", min_available: 1},
-		{label: "event-exporter", min_available: 0},
+		{label: "kube-dns-autoscaler", min_available: 0},
+		{label: "metrics-server", min_available: 0},
 		{label: "kubernetes-dashboard", min_available: 0},
+		{label: "l7-default-backend", min_available: 0},
+		{label: "heapster", min_available: 0},
```

Comment:
Are you sure all of those can be safely restarted? I am rather worried about restarting heapster (or any other critical system pod) in e2e. We had enough pain with rescheduler tainting our nodes already.

Reply:
It's hard to be sure, but:
1. in tests with broken nodes, we don't select a node based on any of those pods, so we sometimes accidentally cause them to become unavailable and rescheduled anyway,
2. we have timeouts and retry logic which should be enough to cover these scenarios if everything works as expected,
3. if it doesn't work as expected, I think we should fail, even if it's not CA that is responsible.

Reply:
I don't necessarily agree with point 3 above. However, we discussed offline with @aleksandra-malinowska and it looks like restarting heapster should no longer break tests.